components/organisms/UserSettingsPage/user-settings-page.tsx

```diff
@@ -316,7 +323,7 @@ const UserSettingsPage = ({ user }: userSettingsPageProps) => {
 
                 <SelectContent position="item-aligned" className="bg-white">
                   {timezones.map((timezone, index) => (
-                    <SelectItem key={`timezone_${index}`} value={timezone.value}>
+                    <SelectItem key={`timezone_${index}`} value={`${timezone.value}_${index}`}>
```

Comment:
Visually this looks fine, but if you look at the generated markup, all the select option values have a postfix of `_some-index`.

![CleanShot 2023-10-25 at 08 41 14](https://github.com/open-sauced/app/assets/833231/80a89af1-489c-4c22-bd61-62e0ef3761a2)

This is the value that is referenced in the database which will not exist.

This will probably work.

```suggestion
                    <SelectItem key={timezone.value} value={timezone.value}>
```

Reply:
Thanks for checking this, Nick.

The warning comes from two duplicate values in the option("GMT Standard Time"). That is why I added the indexes to the end of values and removed the indexes when setting the state of the timezone. 

If you prefer to keep values without indexes, the warning will continue to appear. Please feel free to close this pull request without merging and mark the issue as resolved.

Reply:
No need to close the pull request. I'm sure it can get sorted it you want to pursue this still. 😎

Reply:
I changed my strategy a bit. I gave extra space for the second "GMT StandardTime " in timezones.ts to make it unique, and removed the trailing space when setting the timezoneState. @nickytonline 
![Screenshot from 2023-10-31 16-52-18](https://github.com/open-sauced/app/assets/73622805/58dbf91a-5ea8-474a-8760-69751a289342)

Reply:
Although it works, it's more of a bandaid solution. I haven't had a chance to dig into this, but we should be able to get a unique key generated based on the data.

Reply:
I don't know what else I can do about this issue. I hope you have time to dig into it.